common/common-rest/src/main/java/io/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -126,12 +129,15 @@ protected void scheduleInvocation() {
     });
   }
 
-  protected void runOnExecutor(QueueMetrics metricsData) {
+  protected void runOnExecutor(InvocationStartedEvent startedEvent) {
     Object[] args = RestCodec.restToArgs(requestEx, restOperationMeta);
     createInvocation(args);
 
-    this.invocation.setMetricsData(metricsData);
-    updateMetrics();
+    long startProcessingTime = System.nanoTime();
+    this.invocation.setStartProcessingTime(startProcessingTime);
```

Comment:
can we move this logic to invocation:   invocation.triggerStartProcessingEvent(startedEvent.getStartedTime())
so no need to repeat it in highway

Reply:
Done